### DISCUSSION TITLE
Fixed Group Messaging, Added Testing and CI, and General Improvement

### DISCUSSION THREAD

**Discussion by isFakeAccount** [isFakeAccount/psnawp#19] [General]
**Warning: This version breaks the backward compatibility from the previous versions.**

It has been a while since the last update. I figured maybe I should update the API since there has been some development, thanks to @kerdion. 

Also, I thought I should make this project a bit more proper, like having CI, test, and documentation. The versioning system is changed from cal version to sem version because I thought it conveys important formation more efficiently, Besides, I didn't use the Cal version correctly, to begin with. 

Here is a list of what's changed

- Fixed Group Messaging
- Fixed Search System
- Added CI, Documentation, and Tests
- Made import slightly less awkward
- Fixed the issue where the access_token was fetched on every request instead of using the old one

Future plans:
- Add Trophy support
- Add more endpoints to already existing classes
- Get closer to 100% code coverage

If there are issues with PyPi, let me know, and I'll try to fix them. For now, I have yanked the old versions, but I will remove them in the future. 

<hr /><em>This discussion was created from the release <a href='https://github.com/isFakeAccount/psnawp/releases/tag/v1.0.0'>Fixed Group Messaging, Added Testing and CI, and General Improvement</a>.</em>

**Comment by glebmiller** [comment]
Hi, can you please say if it is possible to get screen shots sent to a group?

**Reply by isFakeAccount** [reply]
Sending pictures to group is possible but I haven't implemented this feature yet.
